conmon-rs/server/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
-name = "conmon-server"
-version = "0.1.0"
+name = "conmon"
+version = "3.0.0"
```

Comment:
are we able to add an rc or anything? we may need to differentiate between the actual final 3.0.0 vs the in-progress one

Reply:
Sure, added a `-dev`